ADR/001: Choosing an API for Cryptocurrency Data Retrieval .md

```diff
@@ -0,0 +1,50 @@
+# ADR: Choosing an API for Cryptocurrency Data Retrieval
+
+**Date**: 31.10.2024
+
+## Context
+The project requires a stable and free access to cryptocurrency market data. The data must be current and include information on real-time prices. We need to consider APIs that provide data without registration or authorization and with rate limits that allow integration into an application without additional costs. The main alternatives considered are the Binance API and CoinGecko API.
+
+## Alternatives
+
+### 1. Binance API
```

Comment:
Does Binance API provides also trading volumes, market capitalization, and historical data same as CoinGecko?

Reply:
In addition to current price information, the api also provides trading volumes, 24-hour changes, order data, and historical data for analyzing price trends.